miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/LibraryDilutionRestController.java

```diff
@@ -126,6 +130,7 @@ public DilutionDto updateDilution(@PathVariable Long dilutionId, @RequestBody Di
       throw new RestException("Cannot convert null to Dilution", Status.BAD_REQUEST);
     }
     LibraryDilution dilution = Dtos.to(dilutionDto);
+    dilution.setCreator(securityStore.getUserByFullName(dilutionDto.getDilutionUserName()));
```

Comment:
Since the creator isn't modifiable, it doesn't need to be loaded from the database for an `update`. (It'll already be fully loaded in the `managed` dilution [here](https://github.com/TGAC/miso-lims/blob/a14d30993b704f603a5ebc27cb0d6fc07d6d3000/miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/impl/DefaultLibraryDilutionService.java#L166) ).